test/stdlib/Inputs/DictionaryKeyValueTypes.swift

```diff
@@ -25,6 +25,31 @@ func equalsUnordered<T : Comparable>(_ lhs: [T], _ rhs: [T]) -> Bool {
   return lhs.sorted().elementsEqual(rhs.sorted())
 }
 
+// A COW wrapper type that holds an Int.
```

Comment:
Is this the right place to define this, and is there an already existing mock COW wrapper I could use here?

Reply:
It's perfectly fine to add this here!